plugins/data-management/src/HierarchicalTrackSelectorWidget/model.js

```diff
@@ -56,13 +56,23 @@ export default pluginManager =>
           return []
         }
 
+        const session = getSession(self)
+        const { assemblyManager } = session
+        const assembly = assemblyManager.get(assemblyName)
```

Comment:
why is the check for initialized needed now after this PR?

Reply:
Good point, it's not actually needed. It just needs to check if `assembly` is defined. I've removed it.